src/www/css/app.css

```diff
@@ -768,10 +788,6 @@ video {
   height: 3rem;
 }
 
-.h-14 {
-  height: 3.5rem;
```

Comment:
Can I just check that these are expected deletions?

Reply:
Here as well.